internal/util/target_test.go

```diff
@@ -149,13 +149,14 @@ var _ = Describe("Target Utilities", func() {
 			Expect(project.Name).To(Equal(testUnreadyProject.Name))
 		})
 
-		It("should require a project or seed when targeting a shoot", func() {
+		It("should a valid shoot when not using a project or seed", func() {
```

Comment:
something is odd with this description, maybe a word is missing

Reply:
✔️